scss/mixins/_grid.scss

```diff
@@ -46,6 +46,7 @@
 @mixin make-col-span($size, $columns: $grid-columns) {
   @if $enable-flex {
     flex: 0 0 percentage($size / $columns);
+    max-width: percentage($size / $columns);
```

Comment:
Assuming the bug doesn't occur in other browsers, this line should have an `// IE10+` comment to note why it's necessary.